deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/circuitbreaker/CircuitBreakerConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkiverse.openapi.generator.deployment.circuitbreaker;
+
+import java.util.List;
+
+public final class CircuitBreakerConfiguration {
+
+    private final List<CircuitBreakerClassConfiguration> classConfigurations;
```

Comment:
I think this can be replaced by 
private final Map<String, List<String> classConfigurations;

Reply:
If you feel that the configuration might contain more thing than a list of method names in the future. Then keep the map (becuase classname is the key to quick access in any case) and change the value to a POJO rather than a list. 
If there is not need for a map at all, because the configuration is processed always as a loop, then the package scoped method should be removed, but I think you need a map, my only doubt is if the value will be a pojo (containing the list of method names and more info) rather than just the list. I vote for using just list now.